playbooks/utils/firewall_update.yml

```diff
@@ -41,6 +41,10 @@
       ansible.builtin.set_fact:
         filter_input: "{{ filter_condition }}{{filter_ip }}"
 
+    - name: gives us all the results
+      ansible.builtin.debug:
+        msg: filter_input
```

Comment:
This var will only have the combination of the condition and the IP - right now that would look like `value contains 128.112.000.00`. If you move this task down under the next task, and use `test_object` as the variable, then you'll see all the results from the firewall.